public/v4/apps/postgres.yml

```diff
@@ -42,7 +44,7 @@ caproverOneClickApp:
             After installation on CapRover, it will be available as srv-captain--YOUR_CONTAINER_NAME at port 5432 to other CapRover apps.
 
             Enter your Postgres Configuration parameters and click on next. It will take about a minute for the process to finish.
-        end: "Postgres is deployed and available as srv-captain--$$cap_appname-db:5432 to other apps. For example with NodeJS: 'const client = new Client({ user: 'cap_pg_user', host: 'srv-captain--$$cap_appname', database: 'cap_pg_db', password: '********', port: 5432})'"
+        end: "Postgres is deployed and available as srv-captain--$$cap_appname:5432 to other apps. For example with NodeJS: 'const client = new Client({ user: '$$cap_pg_user', host: 'srv-captain--$$cap_appname', database: '$$cap_pg_db', password: '********', port: 5432})'"
```

Comment:
I think we can also show the password here instead of `********`.
What do you think?

Reply:
Absolutely. The `********` makes no sense since the password is displayed in plaintext in the environment variables section anyways (which is great!).

Reply:
Good suggestion! It's particularly okay because it's supposed to be the initial password and the user is expected to change post initialization for better security. In fact, after the installation is done, the password can be removed from the env vars and the app works just fine!

Feel free to open a new PR to update this. But for now, merging as is.